plugins/kubernetes-backend/api-report.md

```diff
@@ -187,19 +174,27 @@ export interface KubernetesFetcher {
   ): Promise<PodStatus[]>;
 }
 
-// Warning: (ae-missing-release-tag) "KubernetesObjectsProvider" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
-//
-// @public (undocumented)
+// @alpha (undocumented)
+export interface KubernetesObjectsByEntity {
+  // (undocumented)
+  auth: KubernetesRequestAuth;
+  // (undocumented)
+  entity: Entity;
+}
+
+// @alpha (undocumented)
 export interface KubernetesObjectsProvider {
+  // (undocumented)
+  getCustomResourcesByEntity(
+    customResourcesByEntity: CustomResourcesByEntity,
```

Comment:
```suggestion
    options: CustomResourcesByEntityOptions,
```

What do you think about something like that for both of these? Or some other noun, "input", "request", "query", ...

Reply:
Alright, just had the main-line release so got pretty much a full month to make breaking changes to this addition